src/relayFeeCalculator/chain-queries/queries.e2e.ts

```diff
@@ -20,92 +20,82 @@ dotenv.config();
 describe("Queries", function () {
   test("Arbitrum", async function () {
     const provider = new providers.JsonRpcProvider(process.env.NODE_URL_42161);
-    const arbitrumQueries = new ArbitrumQueries(provider);
+    const arbitrumQueries = new ArbitrumQueries(
+      provider,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      process.env.COINGECKO_PRO_API_KEY
+    );
     await Promise.all([
       arbitrumQueries.getGasCosts("USDC"),
       arbitrumQueries.getTokenDecimals("USDC"),
       arbitrumQueries.getTokenPrice("USDC"),
     ]);
-    const queriesWithCoingeckoProApi = new ArbitrumQueries(
-      provider,
-      SymbolMapping,
-      "0xB88690461dDbaB6f04Dfad7df66B7725942FEb9C",
-      "0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8",
-      "0x893d0d70ad97717052e3aa8903d9615804167759",
-      process.env.COINGECKO_PRO_API_KEY
-    );
-    await queriesWithCoingeckoProApi.getTokenPrice("USDC");
```

Comment:
Why wasn't this correctly testing when the api key is set?

Reply:
Because the `cg` object is only created once by `Coingecko.get()` [here](https://github.com/across-protocol/sdk-v2/blob/master/src/relayFeeCalculator/chain-queries/arbitrum.ts#L33) and [here](https://github.com/across-protocol/sdk-v2/blob/master/src/coingecko/Coingecko.ts#L32) so the first time we create it without the API key